client/src/inside/components/page/EhrElementForm.vue

```diff
@@ -54,6 +55,12 @@
 
 </template>
 
+<style scoped>
+.date-container {
+    min-height: 29vh;
+  }
+</style>
+
```

Comment:
please revert all changes in this file.  I've tried to make this work too and can't see a clean solution.  As is, without these changed, the date picker can be accessed by scrolling the dialog body.  Good enough for a prototype.